Makefile

```diff
@@ -15,3 +15,7 @@ storybook:
 	git config user.email "foo@bar.com"
 	git config user.name "BBC News CI"
 	npm run deploy-storybook;
+
+publish:
+	echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
```

Comment:
Do we want the npm token to be visible in our Jenkins logs?

Reply:
Agreed this is ok for now with @sareh and @ChrisBAshton , will review in JAN

Reply:
```suggestion
	echo "https://registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
```

Reply:
This is following https://blog.npmjs.org/post/118393368555/deploying-with-npm-private-modules so I think should be fine